src/collective/isotope/browser/templates/isotope.pt

```diff
@@ -92,7 +92,7 @@ $(window).load(function(evt) {
                                          item_wf_state item/review_state;
                                          item_wf_state_class python:'state-' + view.normalizeString(item_wf_state);
                                          item_link python:item_type in view.use_view_action and item_url+'/view' or item_url;
-                                         item_has_image python:view.has_image(item);
+                                         item_has_image python:item.getIcon;
```

Comment:
Yes, that is the case.  I was skeptical at first, too, for the poor name choice, but it's for BBB.  See here:

https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/indexers.py#L149